docs/scratch_orgs.rst

```diff
@@ -1,38 +1,172 @@
-Manage scratch orgs
+Manage Scratch Orgs
 ===================
 
-What is an org?
----------------
+Throughout this section, we'll focus on scratch org management in the CumulusCI keychain. To learn about managing persistent orgs, such as sandboxes, production orgs, and packaging orgs, read TODO: reference "Connecting persistent orgs".
```

Comment:
We should probably define what a scratch org is and say a bit about why it's our preferred way to do development.

Reply:
What do you think of the paragraphs added above? I don't want to go into too much detail here, thinking that deeper advocacy probably belongs in other sections, but tried to give a quick overview.